### ISSUE TITLE
request for information: state of anidb support

### ISSUE THREAD

**Issue by OliverLeitner** [nZEDb/nZEDb#2072]
edit site states below the enabling on anidb parsing:

> Whether to attempt to lookup anime information from AniDB. **Currently it is not recommend to enable this.**

is this still the case (has been there as a comment for a while...)?
if so, any chance of getting this "fixed", or if not, have you been looking into alternatives.

it is kind of a bummer, to be unable to choose animes if you dont know them by orig. title, im not japanese/corean...

**Comment by niel** [comment]
I think AniDb lookups ar fine now. I'd never noticed that text before. I'll look into the state of anime lookup and remove that text if appropriate.

**Comment by OliverLeitner** [comment]
hmmm, simply enabling the "anidb parsing all" checkbox didnt do the trick though.

what else do i need to configure to have it starting to parse?

**Comment by OliverLeitner** [comment]
covers dir structure:

```
resources/covers/
.
..
anime
audio
audiosample
book
console
games
movies
music
preview
sample
tvrage
tvshows
video
xxx
```

isnt tvrage dead?
anyways....

all directories are writeable and readable by webserver user. as a result, all other categories are working on creating thumbnails and parsing info.

php error output non existant, same goes for webserver and db server.

**Comment by OliverLeitner** [comment]
some screens visualising the problem:

http://oi66.tinypic.com/2v309qx.jpg
http://oi66.tinypic.com/2aet44y.jpg
http://oi64.tinypic.com/nl4uab.jpg
http://oi68.tinypic.com/64drb4.jpg

and of course, i have the anidb key added to the admin panel site options.

**Comment by niel** [comment]
yes tvrage is dead, but some people still have images in it they might want to use.

**Comment by b3rs3rk** [comment]
@OliverLeitner - you don't put your UDP API Key there.  We don't use the UDP API yet.  You only need the name of the client which is nzedb.  The HTTP API is all we use and they don't require a key for it.  The message was put there a long time ago with the original port from nn+ and never removed because no replay checking was implemented and people were getting IP banned for requesting the same API call.  AniDB is VERY stringent on local caching.  nZEDb now stores a record in the database as to how recently a particular anime was checked for updates and if it is too soon it will skip the API call.  Been running it since I developed it probably a year ago and I haven't had a single ban.
